### PR TITLE
Added description of Asio service

### PR DESCRIPTION
**Pull request by echevemaster**
I'm leaving a short description for asio, I will improve meanwhile we have a big picture of the application

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,7 @@
 # asio
+
+Asio is a dashboard for APIs and software services, track multiple services from one place.
```
Comment (yograterol) [inline README.md]: dashboard?

Comment (echevemaster) [inline README.md]: what word you believe it is fine?

Comment (yograterol) [inline README.md]: Jubulad is our dashboard, for me Asio is a "software services for track apps, services and servers from one place".

**Comment by yograterol** [comment]
@echevemaster make the change and rebase master.

**Pull request merged**